drivers/ios-driver/build.gradle

```diff
@@ -4,25 +4,7 @@ kotlin {
   sourceSets {
```

Comment:
we decided against having an ios-driver artifact and just doing the switch to native-driver, so this module can be removed entirely